app/public/service-worker.js

```diff
@@ -0,0 +1,90 @@
+var reviewRedirect, title, options;
+title = 'We Need Your Help!';
+options = {
+	body: '',
+	icon: '/images/logo-192x192.png',
+	sound: '/sounds/alert.wav',
+	actions: [
+			{
+			action: 'review',
+			title: 'Yes!'
+			}
+		],
+	tag: 'help-notification',
+	vibrate: [500,110,500,110,450]
+	};
+function helpNotification(name,id) {
+	if(name=='404')
+		options.body='';
+	else{
+		reviewRedirect='location/rating/'+id+'/5';
+		options.body='Would you like to add a review to '+name+'?';
+	}
+	//self.registration.showNotification(title, options);
```

Comment:
Does helpNotification do anything after the showNotification was commented out?

Reply:
yes, it will be used to update the data based on the new user locations. I'll rename the methods to clear up the confusion since they've changed from what I originally intended when making them